extensions/amp-story/1.0/amp-story.js

```diff
@@ -403,6 +403,10 @@ export class AmpStory extends AMP.BaseElement {
       this.initializeStandaloneStory_();
     }
 
+    if (this.maybeLoadStoryDevTools_()) {
+      return;
```

Comment:
Should you remove the `amp-story` element from the DOM? `layoutCallback` might get called after this, and all the AMP elements within the Story?

Reply:
Yeah I was testing removing the element locally and it works well. I'm doing it from the maybeLoadStoryDevTools_ method. LMK what you think

Reply:
Oh ok. Let's keep the node in the DOM and apply a `display: none` on the `amp-story` then. That'll prevent all the unbuilt AMP elements from being built. The first two pages of the Story might have already started loading at this point but that's a dev extension that'll load a ton of resources later on, so that's not too big of a deal.
The alternative would be to write some code in the AMP runtime, to short circuit things earlier. We can do that later on if we ever need to optimize things.